packages/style-engine/src/test/index.js

```diff
@@ -308,45 +309,50 @@ describe( 'getCSSRules', () => {
 				value: '5px',
 			},
 			{
-				key: 'fontFamily',
 				selector: '.some-selector',
+				key: 'fontFamily',
```

Comment:
From here on down, it also includes changes to unify the order of properties.

Reply:
Thanks for cleaning this up 🙇